cpp/foxglove/include/foxglove/mcap.hpp

```diff
@@ -61,6 +63,8 @@ struct McapWriterOptions {
   bool repeat_schemas = true;
   /// @brief Whether to truncate the MCAP file.
   bool truncate = false;
+  /// @brief Optional channel filter to use for the MCAP file.
+  SinkChannelFilterFn sink_channel_filter = nullptr;
```

Comment:
This confused me a bit, since I didn't even know that `std::function` (a non-pointer type) could be equal-assigned to `nullptr`. Consider explicitly initializing to `{}` instead.